server/models/chain.ts

```diff
@@ -126,7 +126,11 @@ export default (
         defaultValue: false,
       },
       default_summary_view: { type: dataTypes.BOOLEAN, allowNull: true },
-      snapshot: { type: dataTypes.STRING, allowNull: true },
+      snapshot: {
+        type: dataTypes.ARRAY(dataTypes.STRING),
+        allowNull: true,
+        defaultValue: []
```

Comment:
why do we need the default to be an empty array? could we just keep it NULL?

Reply:
Indeed. Changed!